packages/realm-app-importer/src/AppConfigBuilder.ts

```diff
@@ -18,6 +18,7 @@
 
 export type AppConfig = {
   name: string;
+  baseFilePath?: string;
```

Comment:
This should be removed and we should be using the `AppConfiguration`.  Perhaps this is a good time to refactor the type `AppConfig` into `AtlasAppConfig` or `BaasConfig`.

Reply:
I agree and agree 👍

Reply:
I have refactored it a bit